README.md

```diff
@@ -190,59 +190,98 @@ async def get_user_purchases(user_id: str, period: str = "2021") -> int:
 
 ```
 
-## TTL Policies
+## Composite Indexes and TTL Policies
 
-Firedantic has support for defining TTL policies and creating the policies.
+Firedantic has support for defining composite indexes and TTL policies as well as
+creating them.
+
+### Composite indexes
+
+Composite indexes of a collection are defined in `__composite_indexes__`, which is a
+list of all indexes to be created.
+
+To define an index, you can use `collection_index` or `collection_group_index`,
+depending on the query scope of the index. Each of these takes in an arbitrary amount of
+tuples, where the first element is the field name and the second is the order
+(`ASCENDING`/`DESCENDING`).
+
+The `set_up_composite_indexes` and `async_set_up_composite_indexes` functions are used
+to create indexes.
+
+For more details, see the example further down.
+
+### TTL Policies
 
 The field used for the TTL policy should be a datetime field and the name of the field
 should be defined in `__ttl_field__`. The `set_up_ttl_policies` and
 `async_set_up_ttl_policies` functions are used to set up the policies.
 
-Below are examples (both sync and async) to show how to use Firedantic to set up the TTL
-policies.
-
 Note: The TTL policies can not be set up in the Firestore emulator.
 
-### TTL Policy Example (sync)
+### Examples
+
+Below are examples (both sync and async) to show how to use Firedantic to set up
+composite indexes and TTL policies.
+
+The examples use `async_set_up_composite_indexes_and_ttl_policies` and
+`set_up_composite_indexes_and_ttl_policies` functions to set up both composite indexes
+and TTL policies. However, you can use separate functions to set up only either one of
+them.
+
+#### Composite Index and TTL Policy Example (sync)
 
 ```python
 from datetime import datetime
 
-from firedantic import Model, configure, get_all_subclasses, set_up_ttl_policies
+from firedantic import (
+    collection_index,
+    collection_group_index,
+    configure,
+    get_all_subclasses,
+    Model,
+    set_up_composite_indexes_and_ttl_policies,
+)
 from google.cloud.firestore import Client
 from google.cloud.firestore_admin_v1 import FirestoreAdminClient
 
 
 class ExpiringModel(Model):
     __collection__ = "expiringModel"
     __ttl_field__ = "expire"
+    __composite_indexes__ = [
+        collection_index(('content', "ASCENDING"), ('expire', "DESCENDING")),
```

Comment:
Just thinking that should we maybe also in all places used for indexes also encourage the use of `Query.ASCENDING` instead of `"ASCENDING"` so we'd be consistent with how it's done for the sorting in `.find()`.